components/ModSubmissionLeftNavbar.vue

```diff
@@ -0,0 +1,167 @@
+                @click="scrollToSectionOfFacilityForm(ModSubmissionLeftNavbarSectionIDs.Addresses)"
+            >
+                {{ $t("modPanelFacilitySubmissionLeftNavbar.addresses") }}
+            </button>
+            <button
+                data-testid="submission-form-leftnav-googleMapsInformation"
+                :class="{ 'bg-secondary': activeSection === ModSubmissionLeftNavbarSectionIDs.GoogleMapsInformation,
+                          'bg-primary-inverted': activeSection !== ModSubmissionLeftNavbarSectionIDs.GoogleMapsInformation }"
+                class="w-full py-4 my-2 text-sm text-start pl-2 rounded border-b-2 border-tertiary-bg"
+                @click="scrollToSectionOfFacilityForm(ModSubmissionLeftNavbarSectionIDs.GoogleMapsInformation)"
+            >
+                {{ $t("modPanelFacilitySubmissionLeftNavbar.googleMapsInformation") }}
+            </button>
+            <button
+                data-testid="submission-form-leftnav-healthcareProfessionalIds"
+                :class="{ 'bg-secondary': activeSection === ModSubmissionLeftNavbarSectionIDs.HealthcareProfessionalIds,
+                          'bg-primary-inverted': activeSection !== ModSubmissionLeftNavbarSectionIDs.HealthcareProfessionalIds }"
+                class="w-full py-4 my-2 text-sm text-start pl-2 rounded border-b-2 border-tertiary-bg"
+                @click="scrollToSectionOfFacilityForm(ModSubmissionLeftNavbarSectionIDs.HealthcareProfessionalIds)"
+            >
+                {{ $t("modPanelFacilitySubmissionLeftNavbar.healthcareProfessionalIds") }}
+            </button>
+            <button
+                data-testid="submission-form-leftnav-changeLog"
+                :class="{ 'bg-secondary': activeSection === ModSubmissionLeftNavbarSectionIDs.ChangeLog,
+                          'bg-primary-inverted': activeSection !== ModSubmissionLeftNavbarSectionIDs.ChangeLog }"
+                class="w-full py-4 my-2 text-sm text-start pl-2 rounded border-b-2 border-tertiary-bg"
+                @click="scrollToSectionOfFacilityForm(ModSubmissionLeftNavbarSectionIDs.ChangeLog)"
+            >
+                {{ $t("modPanelFacilitySubmissionLeftNavbar.changeLog") }}
+            </button>
+        </div>
+    </div>
+</template>
+
+<script setup lang="ts">
+import { ref, type Ref, onMounted, onUnmounted } from 'vue'
+import { useModerationSubmissionsStore } from '~/stores/moderationSubmissionsStore'
+import { ModSubmissionLeftNavbarSectionIDs } from '~/stores/moderationScreenStore'
+import { handleScroll, type SectionInformation } from '~/utils/handleScroll'
+
+const moderationSubmissionsStore = useModerationSubmissionsStore()
+const activeSection: Ref<string> = ref(ModSubmissionLeftNavbarSectionIDs.ContactInformation)
+const isScrolling: Ref<boolean> = ref(false)
+
+const scrollToSectionOfFacilityForm = (sectionId: ModSubmissionLeftNavbarSectionIDs) => {
```

Comment:
you know what you did here. make the things optional like you were thinking and said on discord😆